security/README.md

```diff
@@ -0,0 +1,42 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Passwords
+---------
+
+* Use a unique password for every account you create.
+* Don't store passwords in plain text.
```

Comment:
> I think these should be dev-focused, so I'd kill the first bullet.

I think it's really important that developers practice good personal security. Otherwise, attackers can use a developer's carelessness to circumvent good security on deployed services.

I'd like to have this guide contain enough personal security recommendations that developers don't compromise the applications that they're deploying. Basic password recommendations seem like an easy place to start.